src/DynamoCoreWpf/ViewModels/PackageManager/PublishPackageViewModel.cs

```diff
@@ -1335,15 +1335,28 @@ private void RemoveItem(object parameter)
         {
             if (!(parameter is PackageItemRootViewModel packageItemRootViewModel)) return;
 
-            if (packageItemRootViewModel.FileInfo == null || packageItemRootViewModel.FileInfo == null)
+            string fileName = packageItemRootViewModel.FileInfo == null ? packageItemRootViewModel.Name : packageItemRootViewModel.FileInfo.FullName;
+            string fileType = packageItemRootViewModel.DependencyType.ToString();
+
+            if (fileName.ToLower().EndsWith(".dll") || fileType.Equals(DependencyType.Assembly))
```

Comment:
If the filetype is DependencyType.Assembly, I suppose it should be a dll. Do we need both the checks?

Reply:
Yes, this was the case before, but I noticed that we do not get filename with `.dll` extension, therefore added another check that will satisfy this condition, putting an `&&` will not pass if the filename is without extension.

Reply:
what about packages that ship .exes?

Reply:
According to the current flow, they will end up in additional files

Reply:
Do you want me to add `.exe` extension to assemblies?

Reply:
Do exes have Dependency Type as Assembly?

Reply:
I think not - according to:
https://github.com/DynamoDS/Dynamo/blob/74709f73df1b87a1d4d3997da947dac3dea3697f/src/DynamoPackages/Package.cs#L335

we only scan files ending in `*.dll` and assign those types as assembly. I guess like you said above .exes would just be another file in extra.

Reply:
I published a test package with exes, it ends up in additional files, so we are good.
thanks you for the review @reddyashish @mjkkirschner